health/health_test.go

```diff
@@ -0,0 +1,161 @@
+package health
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"testing"
+
+	"github.com/featureform/metadata"
+	pc "github.com/featureform/provider/provider_config"
+	pt "github.com/featureform/provider/provider_type"
+	"github.com/joho/godotenv"
+	"go.uber.org/zap/zaptest"
+)
+
+var providerType = flag.String("provider", "", "provider type under test")
+
+type testMember struct {
+	providerDef metadata.ProviderDef
+}
+
+func checkEnv(envVar string) string {
+	value, has := os.LookupEnv(envVar)
```

Comment:
@ahmadnazeri I remember ahmad saying we need to not fail for tests because end2end tests don't have an env or something. I can't quite remember

Reply:
I am honestly not sure what this is referring to